server/routes/token-exchange.js

```diff
@@ -46,24 +47,19 @@ router.post('/', (req, res) => {
                     console.log("getting userinfo");
                     if (!error) {
                         res.send({user: JSON.parse(body)});
+                    } else {
+                      res.sendStatus(503);
```

Comment:
Is that the correct error code?

Can we add documentation to the readme specifying that?

Reply:
Actually, seems 500 seems appropriate: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500 because the server is misconfigured (it should be configured to have both refresh and access tokens, but FusionAuth might not be). 

Fight me. :)

Reply:
503 tells me that the server is "down" and I am gonna go run to the sysadmin to get it back up. 500 will just tell me something exploded and I will go hunt the logs for what happened.

Also is the server still dying when errors happen? we might want this in a try catch

Reply:
Agree that this case is likely a misconfiguration, so changed to 500.  Left the other error case as 503, because that indicates a more transient error state (which happens with duplicate requests).

Generally, we should have more robust error handling for various response codes.  I'll file an issue.

Reply:
thanks!